src/Native/Runtime/arm/AllocFast.S

```diff
@@ -4,4 +4,60 @@
 
 #include <unixasmmacros.inc>
 
+.syntax unified
+.thumb
+
 // TODO: Implement Arm support
+
+// Allocate non-array, non-finalizable object. If the allocation doesn't fit into the current thread's
+// allocation context then automatically fallback to the slow allocation path.
+//  r0 == EEType
+    LEAF_ENTRY RhpNewFast, _TEXT
+    LEAF_END RhpNewFast, _TEXT
+
+// Allocate non-array object with finalizer.
+//  r0 == EEType
+    LEAF_ENTRY RhpNewFinalizable, _TEXT
```

Comment:
It would be nice to make the implementation breakpoint to get nice crash for unimplemented parts once you start running the programs.